src/ImageSharp/Formats/Webp/Lossless/PredictorEncoder.cs

```diff
@@ -572,19 +551,17 @@ private static byte NearLosslessComponent(byte value, byte predict, byte boundar
 
                 return (byte)lower;
             }
-            else
-            {
-                // upper is closer to residual than lower.
-                if (residual <= boundaryResidual && upper > boundaryResidual)
-                {
-                    // Halve quantization step to avoid crossing boundary. This midpoint is
-                    // on the same side of boundary as residual because midpoint <= residual
-                    // (since upper is closer than lower) and residual is below the boundary.
-                    return (byte)(lower + (quantization >> 1));
-                }
 
-                return (byte)(upper & 0xff);
+            // upper is closer to residual than lower.
+            if (residual <= boundaryResidual && upper > boundaryResidual)
+            {
+                // Halve quantization step to avoid crossing boundary. This midpoint is
+                // on the same side of boundary as residual because midpoint <= residual
+                // (since upper is closer than lower) and residual is below the boundary.
+                return (byte)(lower + (quantization >> 1));
             }
+
+            return (byte)(upper & 0xff);
```

Comment:
```suggestion
            return (byte)upper;
```
With the cast to `byte` to mask isn't necessary. But modern JIT will optimize this away anyway. It's solely about less IL-size (if that counts).